inventory_refresh.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", "~> 5.0"
```

Comment:
Can we apply https://github.com/ManageIQ/activerecord-id_regions/pull/18/files#r537735807 here as well?

Reply:
Yup, agreed.  Since this is a gem we publish, and there is seemingly no other change, just supporting Rails 5 and 6 should be sufficient for our purposes.

Reply:
Not sure we can do this now, since there is a `.update_attributes` change I did.  I probably can remove that change, but wanted to get everyone else's opinion before I do.

Reply:
@NickLaMuro it looks like the only changes from `#update_attributes` to `#update` were in specs, I'm good with `">=5.0", "< 6.1"`

Reply:
Good point.  I think I saw ` lib/inventory_refresh/inventory_collection/helpers/initialize_helper.rb` in the files changed diff, and didn't take a look at the contents, which only changes a comment and is fine.